tests/002-test-reject-disk-with-lvm-signature.sh

```diff
@@ -9,11 +9,12 @@ test_lvm_sig() {
   local vg_name="dss-test-foo"
 
   # Error out if any pre-existing volume group vg named dss-test-foo
-  if vg_exists "vg_name"; then
+  if vg_exists "$vg_name"; then
```

Comment:
All related changes are generally kept in one patch.

So one good way to organize this patch series would be.

- First patch fixes this vg_name issue
- Second patch fixes cleanup config file issue across remaining tests.
- Third patch fixes test_status issue across all tests.